app/views/admin/users/new.html.haml

```diff
@@ -5,22 +5,7 @@
         %legend.govuk-fieldset__legend.govuk-fieldset__legend--xl
           %h1.govuk-fieldset__heading
             Add a new user
-        - if @user.errors.present?
-          .govuk-error-summary
-            %h2#error-summary-title.govuk-error-summary__title
-              There is a problem
-            .govuk-error-summary__body
-              %ul.govuk-list.govuk-error-summary__list
-                - @user.errors.full_messages.each do |message|
-                  %li= message
-        .govuk-form-group
```

Comment:
I'm assuming these =form.label etc should have been deleted in the previous commit, not here?

Reply:
Yes it should.